libs/api/src/main/java/de/cyface/api/ClassifiedSegmentRetriever.java

```diff
@@ -37,6 +37,7 @@
 import io.vertx.core.json.JsonObject;
 import io.vertx.ext.mongo.MongoClient;
 
+// TODO: This should probably be deleted or go into some other project.
```

Comment:
Please see here.

Reply:
I'm moving `libs/api` to an own (public) repo, see https://cyface.atlassian.net/browse/RFR-23